### DISCUSSION TITLE
[Ata da reuniao] 3 de maio de 2021, 21h

### DISCUSSION THREAD

**Discussion by jenrodrigues** [andrebaltieri/AgendaEventos#47] [General]
### **Quem participou:**
Balta
Gabriel Nalin
Brewerton Santos
Jenifer Rodrigues
Joao Colussi
Luan Freitas
Nilson Teodoro
Rene Bentes

### **Topicos debatidos:**
1) Apresentamos pro Balta os cards que foram escolhidos pelo time pra coluna Commited.
2) Apresentamos tambem as propostas do modelo de entidade relacionamento. Algumas modificacoes foram sugeridas pelo Balta.
3) Definimos o backlog dessa semana e nos distribuimos entre as issues.

### **Proximos passos:**
- Revisar DER, MER e diagrama de classes. @jenrodrigues e @BrewertonSantos 
- Criar console app no VSCode com a estrutura de classes proposta. @renebentes, @nilson-teodoro, @colussicode e @LuanFreitasRibeiro 
- Se der tempo, realizar a issue “Criar os scripts da base de dados”. **Atribuicao pendente.**

### **Proxima reuniao em 10 de maio as 21h. Topicos a serem debatidos:**
- Apresentar ao Balta a console app com as classes do projeto.
- Caso tenha sido feito, apresentar ao Balta o script de criacao da base de dados.

**Comment by BrewertonSantos** [comment]
Penso que como próxima reunião deve estar marcando a reunião da quarta (05/05/2021 21hrs  BRT). Onde realinhamos e discutimos progressos antes da reunião das segundas com o Balta.

**Reply by jenrodrigues** [reply]
Vixe, eu nao havia previsto essa reuniao de quarta-feira. Vou marcar o pessoal aqui pra checar quem poderia comparecer. 
@renebentes @nilson-teodoro @colussicode @LuanFreitasRibeiro @JoaoNascSilva

**Reply by JoaoNascSilva** [reply]
Bom dia, Tudo bem, marcado aqui Quarta(05/05/2021) as 21:00

**Reply by renebentes** [reply]
Boa tarde. Agendado.

**Reply by GabrielNalin** [reply]
Boa noite, não fui marcado mas estarei presente😁
